sdk/include/reactos/ui/branding.h

```diff
@@ -0,0 +1,594 @@
+/*
+ * PROJECT:     ReactOS headers
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Branding support for dialog boxes.
+ * COPYRIGHT:   Copyright 2006 Hervé Poussineau <hpoussin@reactos.org>
+ *              Copyright 2019 Stanislav Motylkov <x86corez@gmail.com>
+ *              Copyright 2020 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ *              Copyright 2022 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
+ */
+
+#pragma once
```

Comment:
I see you started to move branding into SDK. Is it possible to move actual resources (bitmaps) also there to avoid duplicating them in source tree?

Reply:
This could be interesting to do. But then it would be useful to move them into the "media" folder... as long as they are indeed duplicated (aka. the very same files).